wdn/templates_4.1/scripts/navigation.js

```diff
@@ -665,6 +665,10 @@ define(['jquery', 'wdn', 'modernizr', 'require'], function($, WDN, Modernizr, re
 						switchSiteNavigation($homepageCrumbLink[0], false);
 					});
 
+					$(navSel + ' a').last().focusout(function(){
```

Comment:
It's a _trivial_ amount of overhead; events are always bubbling and the more events we can condense into a single element reduces the memory footprint.